docs/.vuepress/config.js

```diff
@@ -8,7 +8,12 @@ module.exports = {
     lineNumbers: true
   },
   themeConfig: {
-    lastUpdated: "Last Updated",
+    lastUpdated: true,
+    algolia: {
+      apiKey: 'a6e2f64347bb826b732e118c1366819a',
```

Comment:
Do we really want to put our Algolia API key in plain text here???

Reply:
yup it's fine, we did the same for tendermint, Jessy approved, and:
https://www.algolia.com/doc/guides/security/api-keys/?language=php#search-only-api-key

Reply:
Oh! Nice! thank you! I'll merge this in!